core/o11y/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
can we make these two a symbolic link? I think tahth should work

Reply:
Ah yeah, could've sworn I did `ln`, apparently it was `cp`, lol. Fixing.